### PR TITLE
Added support for image copy path

### PR DESCRIPTION
**Pull request by drpayyne**
**Description** (*)

Added support for images in copy magento path action.

**Fixed Issues (if relevant)**

1. Fixes magento/magento2-phpstorm-plugin#414

**Questions or comments**

**Contribution checklist** (*)
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [x] All new or changed code is covered with integration/functional tests (if applicable)
 - [x] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Comment by DmitryFurs** [comment]
@drpayyne Can you add support for **svg** and **webp** files? Perhaps you should review this [section](https://github.com/magento/magento2/blob/2.4-develop/pub/media/.htaccess#L39-L74) and add some more supported file types. Thanks!

File: src/com/magento/idea/magento2plugin/actions/CopyMagentoPath.java [hunk 1]
```diff
@@ -22,10 +22,8 @@
 
 public class CopyMagentoPath extends CopyPathProvider {
     public static final String PHTML_EXTENSION = "phtml";
-    public static final String JS_EXTENSION = "js";
-    public static final String CSS_EXTENSION = "css";
-    private final List<String> acceptedTypes
-            = Arrays.asList(PHTML_EXTENSION, JS_EXTENSION, CSS_EXTENSION);
+    public static final List<String> WEB_EXTENSIONS
+            = Arrays.asList("js", "css", "png", "jpg", "jpeg", "gif");
```
Comment (eduard13) [inline src/com/magento/idea/magento2plugin/actions/CopyMagentoPath.java:26]: Just wondering, isn't possible to use [ImageIO::read](https://docs.oracle.com/javase/7/docs/api/javax/imageio/ImageIO.html#read%28java.io.File%29)? This should return `null` if the provided file cannot be read as an image. Or maybe there are some alternatives to check the file's `contentType`.

Comment (drpayyne) [reply src/com/magento/idea/magento2plugin/actions/CopyMagentoPath.java:26]: I'm currently thinking of a way to support ONLY the Magento supported types. There's a lot more image types I missed which @DmitryFurs has described. So I'll check if the ImageIO types match the supported types, and develop a better solution. Thanks for the suggestion @eduard13 and @DmitryFurs!

**Comment by drpayyne** [comment]
Apologies, got occupied with other work. Will try to work on this over this weekend.

**Comment by bohdan-harniuk** [comment]
Hello, @drpayyne!

Could you please change the base branch to the 4.3.0-develop?
And, what about current progress?
Did you finish everything here or some changes should be done in addition?

Regards,

**Comment by bohdan-harniuk** [comment]
Hello, @drpayyne!

Any updates here?

Regards,

**Comment by drpayyne** [comment]
Hi @bohdan-harniuk, unfortunately I'm held up with work and won't be able to get this PR to completion as it's been quite a while too. If anyone can continue this PR, that'd be great. Thank you!

**Comment by bohdan-harniuk** [comment]
Hello, @doninAtwix!

Please, take a look at this PR.
Check if you can proceed with rest of development here.

Regards,

**Comment by bohdan-harniuk** [comment]
Hello, @drpayyne!

Thank you for your work here!
@doninAtwix has finished this task in the https://github.com/magento/magento2-phpstorm-plugin/pull/1020.

Regards,

**Pull request closed without merging**